provider/internal/device/ios/data_source.go

```diff
@@ -83,6 +87,14 @@ func (d *IosDataSource) Schema(ctx context.Context, req datasource.SchemaRequest
 				MarkdownDescription: "This attribute indicates whether certificates were ignored when onboarding this device.",
 				Computed:            true,
 			},
+			"tags": schema.ListAttribute{
```

Comment:
I was just reading the CDO documentation: we call tags labels in the documentation. So let's call them labels here in the provider (let's continue to call them tags in the client, because that's what they are internally).

Could you also please change the markdown description to: "The labels applied to the device. Labels are used to group devices in CDO. Refer to the [CDO documentation](https://docs.defenseorchestrator.com/t-applying-labels-to-devices-and-objects.html#!c-labels-and-filtering.html) for details on how labels are used in CDO."

Reply:
Sure done